tests/unit/factories.py

```diff
@@ -0,0 +1,54 @@
+import factory
+from factory.alchemy import SQLAlchemyModelFactory
+
+from ctms import models
+from ctms.database import ScopedSessionLocal
+
+
+class BaseSQLAlchemyModelFactory(SQLAlchemyModelFactory):
+    class Meta:
+        abstract = True
+        sqlalchemy_session = ScopedSessionLocal
+
+
+class NewsletterFactory(BaseSQLAlchemyModelFactory):
+    class Meta:
+        model = models.Newsletter
+
+    name = factory.Sequence(lambda n: f"newsletter-{n}")
+    subscribed = True
+    format = "T"
+    lang = factory.Faker("language_code")
+    source = factory.Faker("url")
+
+    email = factory.SubFactory(factory="tests.unit.factories.EmailFactory")
+
+
+class EmailFactory(BaseSQLAlchemyModelFactory):
+    class Meta:
+        model = models.Email
+
+    email_id = factory.Faker("uuid4")
+    primary_email = factory.Faker("email")
+    basket_token = factory.Faker("uuid4")
+    first_name = factory.Faker("first_name")
+    last_name = factory.Faker("last_name")
+    mailing_country = factory.Faker("country_code")
+    email_format = "T"
+    email_lang = factory.Faker("language_code")
+    double_opt_in = False
+    has_opted_out_of_email = False
```

Comment:
Do we need to modify this for active/inactive contacts? _(I think active contacts should have `double_opt_in` to True)_

Reply:
I believe we will have to override this default for different test cases -- either we keep this as is and override `double_opt_in` where applicable, or vice versa. 

We could also create additional factories with different defaults for different test cases.